docs/Userguide_datasets.rst

```diff
@@ -31,7 +31,7 @@ Mila hosts/seeds some datasets created by the Mila community through `Academic
 Torrent <https://academictorrents.com>`_. The first step is to create `an
 account and a torrent file <https://academictorrents.com/upload.php>`_.
 
-Then drop the dataset in ``/miniscratch/transit_datasets`` and send the
+Then drop the dataset in ``/network/scratch/transit_datasets`` and send the
```

Comment:
Should we hide the directory?  `/network/scratch/.transit_datasets`
It has a specific purpose which is defined here. A copy & paste of the path should be ok.

Reply:
I believe we chose the hidden approach right (`/network/scratch/.transit_datasets`)? I'm ok with both

Reply:
Indeed, I updated the change to reflect our decision.